pkg/helm/fake/fake_actions.go

```diff
@@ -0,0 +1,30 @@
+package fake
```

Comment:
you have introduced here a package that could be accessed and used from the production code as well, and IMHO we should avoid that. Thus, I propose that we move the content of this file in `handler_test.go`. This allows us as well to make all functions private.